qiskit/circuit/instruction.py

```diff
@@ -456,7 +465,9 @@ def inverse(self):
         inverse_definition = self._definition.copy_empty_like()
         inverse_definition.global_phase = -inverse_definition.global_phase
         for inst in reversed(self._definition):
-            inverse_definition._append(inst.operation.inverse(), inst.qubits, inst.clbits)
+            inverse_definition._append(
+                inst.operation.inverse(annotated=annotated), inst.qubits, inst.clbits
```

Comment:
This doesn't hurt but if `annotated=True` this line would never get run would it? As we return early if `annotated` is true at L442.

Reply:
Removed it.